doc/flexbench/docs/GSoC/2024.md

```diff
@@ -0,0 +1,92 @@
+# 2024
+
+**Mentors**: Giannis Skitsas, Marios Karagiannopoulos
+
+**Assignee**: Yujun Liu
+
+## Abstract
+
+Flexbench was a forked project from the http-traffic-simulator npm package that has evolved significantly. The initial development during GSoC 2022 focused on transforming the package into a stand-alone desktop application and a web server with API exposure, supporting authentication and authorization through OpenIDC, and deploying it as a containerized application on Kubernetes. my work aims to further enhance Flexbench by implementing a anonymization feature to secure sensitive user data during testing, integrating OpenAPI to streamline and standardize API interactions, and addressing various UI/UX improvements for destop-app to ensure a more intuitive user experience. 
+
+## Main goals for GSoC 2024
+
+The three main goals for this summer are-
+
+- Enhance and fix aesthetic and UI/UX bugs
+- Research and Implement anonymization feature for core lib
+- OpenAPI Support Integration
+
+
+<details>
+<summary>Anonymization Feature Report</summary>
+
+[Anonymization Feature Report](Anonymization-Feature-Report.md)
+
+</details>
+
+<details>
+<summary>OpenAPI Integration Report</summary>
+
+[OpenAPI Integration Report](OpenAPI-Integration-Report.md)
+
+</details>
+
+## Implementation 
+
+### Destop-app
+
+- Finding the bugs for aesthetic and UI/UX (S1)
```

Comment:
What is (S1)?

Reply:
S1 is something I picked up from the format of the last contributor in 2022.md, which matches the PR at the bottom, I will remove it if it is confusing.